haystack/testing/document_store.py

```diff
@@ -6,8 +6,6 @@
 from datetime import datetime
 from typing import List
 
-import pandas as pd
```

Comment:
The changes in this file can be better understood by reading https://github.com/deepset-ai/haystack/pull/8684